hud/DaysPassedTracker.gd

```diff
@@ -0,0 +1,9 @@
+extends HBoxContainer
+
+@onready var database = get_node("/root/Database")
+
+func _ready():
+	$NumberOfDays.text = str(database.day_count)
+
+func _process(delta):
+	$NumberOfDays.text = str(database.day_count)
```

Comment:
For future reference, I don't think we want to put this kind of assignment into _process, it's an expensive way to do reflection. It's not a hard refactor for the future, I think we'll just want it to be event driven / called by something that's handling day advancement.

Reply:
Fixed via [6696b21](https://github.com/SamBumgardner/packrat-game/pull/1/commits/6696b2192ea3d9b1c4f1da7434c1318879aeea89).